commit e4025b2911d30bd2e038096316448a6e68942743 (refs/changes/01/2501/1)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-09-06T11:28:10-04:00 (2 years, 1 month ago)
    
    ZAPI-798 moray bucket initialization integration test fails due to recent moray changes

diff --git a/test/vms.update-moray-bucket-non-transient-error.test.js b/test/vms.update-moray-bucket-non-transient-error.test.js
index 694c48a..d1c132c 100644
--- a/test/vms.update-moray-bucket-non-transient-error.test.js
+++ b/test/vms.update-moray-bucket-non-transient-error.test.js
@@ -164,7 +164,8 @@ exports.moray_init_non_transient_error = function (t) {
 
         vmapiClient.ping(function onVmapiPing(pingErr, obj, req, res) {
             var errBody = pingErr.body;
-            var expectedErrString = 'docker.type is invalid';
+            var expectedErrString = 'bucket.index[\'docker\'].type should be ' +
+                'equal to one of the allowed values';
             var expectedHealthiness = false;
             var expectedResponseHttpStatus = 503;
             var expectedStatus = 'some services are not ready';
